src/applications/mhv/medical-records/containers/Allergies.jsx

```diff
@@ -125,6 +125,40 @@ const Allergies = props => {
     makePdf(pdfName, pdfData, 'Allergies', runningUnitTest);
   };
 
+  const generateAllergiesTxt = async () => {
+    const product = `
+    Allergies and reactions \n 
+
+    Review allergies, reactions, and side effects in your VA medical
+    records. This includes medication side effects (also called adverse drug
+    reactions). \n
+
+    If you have allergies that are missing from this list, tell your care
+    team at your next appointment. \n
+    
+    Showing ${allergies.length} from newest to oldest. \n
+
+    ${allergies.map(
+      entry =>
+        `_____________________________________________________ \n
+      ${entry.name} \n
+      \t Date entered: ${entry.date} \n
+      \t Signs and symptoms: ${entry.reaction} \n
+      \t Type of Allergy: ${entry.type} \n
+      \t Location: ${entry.location} \n
+      \t Observed or historical: ${entry.observedOrReported} \n
+      \t Provider notes: ${entry.notes} \n`,
+    )}`;
+
+    const blob = new Blob([product], { type: 'text/plain' });
```

Comment:
It doesn't have to be in this PR, but we'll probably want to move lines 153 through 159 to the PrintDownload component so that the code isn't repeated in every component that uses text download.